pipelines/controller/src/main/java/com/google/fhir/analytics/DwhFilesManager.java

```diff
@@ -62,9 +64,12 @@ public class DwhFilesManager {
 
   private final DataProperties dataProperties;
 
+  @SuppressWarnings(
+      "NullAway") // Suppressing warning because the field is initialized in init() and includes a
```

Comment:
nit: please move the comment to the line above before the annotation; this looks a little bit odd.